multiqc/modules/multivcfanalyzer/multivcfanalyzer.py

```diff
@@ -0,0 +1,244 @@
+        }
+        headers['filteredVarCall'] = {
+            'title': 'Filtered SNP Call',
+            'description': 'Number of positions ignored defined in user-supplied filter list',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['refCall'] = {
+            'title': 'Number of Reference Calls',
+            'description': 'Number of reference calls made',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['discardedRefCall'] = {
+            'title': 'Discarded Reference Call',
+            'description': 'Number of reference positions not reaching genotyping quality threshold',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'calls'
+        }
+        headers['noCall'] = {
+            'title': 'Positions with No Call',
+            'description': 'Number of positions with no call made as reported by GATK',
+            'scale': 'BuPu',
+            'shared_key': 'calls'
+        }
+        headers['coverage (fold)'] = {
+            'title': 'SNP Coverage',
+            'description': 'Average number of reads covering final calls',
+            'scale': 'OrRd',
+            'shared_key': 'coverage',
+            'suffix': 'X'
+        }
+        headers['coverage (percent)'] = {
+            'title': '% SNPs Covered',
+            'description': 'Percent coverage of all positions with final calls',
+            'scale': 'PuBuGn',
+            'shared_key': 'coverage',
+            'suffix': '%'
+        }
+        headers['unhandledGenotype'] = {
+            'title': 'Unhandled Genotypes',
+            'description': 'Number of positions discarded due to presence of more than one alternate allele',
+            'scale': 'BuPu',
+            'hidden': True,
+            'shared_key': 'snp_count'
+        }
+
+        #Separate table config
+        table_config = {
+        'namespace': 'MultiVCFAnalyzer',                         # Name for grouping. Prepends desc and is in Config Columns modal
+        'id': 'mvcf-table',                 # ID used for the table
+        'table_title': 'MultiVCFAnalyzer Results',             # Title of the table. Used in the column config modal
+        }
+        tab = table.plot(self.mvcf_data, headers, table_config)
+        return tab
+
+
+    
+    def addBarplot(self):
+        """ Take the parsed stats from MultiVCFAnalyzer and add it to the MVCF Table"""
+        cats = OrderedDict()
+
+        cats['SNP Calls (all)'] = {
+            'name': 'SNP Calls (all)',
+            'color': '#8bbc21'
+        }
+        cats['discardedVarCall'] = {
+            'name': 'Discarded SNP Call',
+            'color': '#f7a35c'
+        }
+        cats['filteredVarCall'] = {
+            'name': 'Filtered SNP Call',
+        }
+        cats['refCall'] = {
+            'name': 'Number of Reference Calls',
```

Comment:
Just `Reference Calls'

Reply:
Review tips:

* Group comments in to a single review
* Use code suggestion blocks

@apeltzer will thank you 😉 

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request

Reply:
Oops my bad. Got carried away.

Reply:
No problem :) Note that all of your new code suggestions you just added were as individual reviews. You can do multiple comments (each with a code suggestion) under a single review next time 👍

You can see the difference by comparing how this review page looks in contrast to something like https://github.com/ewels/MultiQC/pull/1009#pullrequestreview-303728509 where they are grouped..

This looks a little cleaner and also means that everyone on the thread gets one email, instead of 20 😉 

---

![image](https://user-images.githubusercontent.com/465550/69234842-6c655d80-0b90-11ea-9134-08a19188388b.png)

---

![image](https://user-images.githubusercontent.com/465550/69234878-7f782d80-0b90-11ea-8158-86f122e89541.png)

Reply:
Christ! Sorry, I was adding the code suggestions as just 'add single comment' so I had assumed they weren't coming as a review. 

I'll find somewhere less public to practise... 😅 Thanks for the patience!